src/admin-views/attendees.php

```diff
@@ -110,7 +110,11 @@
 							<li>
 								<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
 								echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
-								?></li>
+
+								$this->set( 'ticket_item_for_overview', $ticket );
+								$this->do_entry_point( 'overview_section_after_ticket_name' );
```

Comment:
Let's add a phpdoc block like we do with normal actions, so a `@version` and description

Reply:
`@since` I meant

Reply:
Thanks. Updated.